boards/arduino/portenta_c33/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Rahul Arasikere <arasikere.rahul@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_ARDUINO_PORTENTA_C33
+
+if NETWORKING
+
+config NET_L2_ETHERNET
+	default y
+
+endif # NETWORKING
+
+#if BOARD_ARDUINO_PORTENTA_C33
+#source "boards/common/usb/Kconfig.cdc_acm_serial.defconfig"
+#endif
```

Comment:
???

have seen it is edited in later commit, squash the board and board usb commit together